docker-app/qfieldcloud/core/admin.py

```diff
@@ -1038,6 +1038,17 @@ def has_change_permission(self, request, obj):
         return False
 
 
+class ActiveUsersFilter(admin.SimpleListFilter):
+    title = _("active_users")
+    parameter_name = "active_users"
+
+    def lookups(self, request: Any, model_admin: Any) -> List[Tuple[Any, str]]:
+        return [("is_active", _("is active"))]
+
+    def queryset(self, request, qs: QuerySet) -> QuerySet:
+        return qs.filter(useraccount__current_subscription_vw__isnull=False)
```

Comment:
I am not quite sure about what the ticket wants, but I guess it is more about active members? Or active owners? Maybe a list of active members in the detail view? I added a comment in the clickup task.

Regarding the filter: With this query I guess the naming should be similar to the People filter, either adjust one or the other:
![image](https://github.com/opengisch/qfieldcloud/assets/21113500/873bac17-dd69-4507-8266-cfe7f9a88092)

Reply:
Thanks for the inputs, after discussing with Ivan the specs have become much clearer. I'll ping you when it's ready.

Reply:
Sorry for the vague task definition, my fault!